src/lib.rs

```diff
@@ -152,28 +152,36 @@
 //! LD_LIBRARY_PATH=/path/to/mylib/target/debug`. Now, you should get the
 //! expected output `Hello, josh!` from your Java class.
 //!
-//! If you build with the `invocation` feature, there is yet another dependency:
-//! `libjvm` (`libjvm.so`, `libjvm.dylib`, `jvm.dll` depending on target platform).
+//! # Launching JVM from Rust
 //!
-//! During build time jni-rs checks the `JAVA_HOME` environment variable
-//! and tries to find there libjvm. If this failes, jni-rs tries to ask java
-//! (`java -XshowSettings:properties -version 2>&1 | grep 'java.home'`).
-//! On Windows additionally needed path to `jvm.lib`.
+//! If you need to use the part of the [Invocation API]
+//! that allows to launch a JVM
+//! from a native process, you must enable `invocation` feature.
+//! The application will require linking to the dynamic `jvm`
+//! library, which is distributed with the JVM.
 //!
-//! If you encountered error messages when building, first check your `JAVA_HOME`
-//! and make sure it points to a correct JDK location.
+//! During build time, the JVM installation path is determined:
+//! 1. By `JAVA_HOME` environment variable, if it is set.
+//! 2. Otherwise — from `java` output.
 //!
-//! At runtime a binary executable or a dynamic library authomatically tries
-//! to load and link to JVM at startup. As with `mylib` above, the path to JVM
-//! should be configured with `LD_LIBRARY_PATH`/`PATH`, depending on platform.
+//! For more information - see documentation in [build.rs](https://github.com/jni-rs/jni-rs/tree/master/build.rs).
```

Comment:
I'd move this sentence to the end — if someone have read the section and needs further info.